src/ngraph/op/fused/squeeze.cpp

```diff
@@ -131,3 +132,49 @@ shared_ptr<Node> op::Squeeze::clone_with_new_inputs(const OutputVector& new_args
     }
     return make_shared<Squeeze>(new_args.at(0), new_args.at(1));
 }
+
+namespace
+{
+    template <element::Type_t ET>
+    bool evaluate(const HostTensorPtr& arg0, const HostTensorPtr& out)
+    {
+        runtime::reference::copy(
+            arg0->get_data_ptr<ET>(), out->get_data_ptr<ET>(), shape_size(out->get_shape()));
+        return true;
+    }
+
+    bool evaluate_squeeze(const HostTensorPtr& arg0, const HostTensorPtr& out)
+    {
+        bool rc = true;
```

Comment:
Need to set the output element type and shape here in case the input was dynamic.